libs-scala/ports/BUILD.bazel

```diff
@@ -4,6 +4,7 @@
 load(
     "//bazel_tools:scala.bzl",
     "da_scala_library",
+    "da_scala_test",
 )
 
 da_scala_library(
```

Comment:
Do we not need maven tags on this? It looks almost impossible that this is not pulled in by any artifact that we publish to maven now.

Reply:
Somewhat related, if this is not just internal but published to Maven, maybe there is some library for this that we could use instead?

Reply:
Good call. I don't think there is anything out there that I've seen; it's almost too trivial to publish. I shall add the Maven tags though.

Is there any way to run the Maven tag checks locally? Couldn't find a simple way to do it. It'd be nice if it was a test case that was run with `bazel test //release/something`.

Reply:
Running `release.sh` should do the trick although I think with the new release process this might  end up rebuilding the world.

Reply:
Yeah, pass. I'll add it to my backlog of things to do once I have nothing else to do.